src/charm.py

```diff
@@ -184,6 +184,21 @@ def _gen_pebble_plan(self) -> ops.pebble.LayerDict:
                     },
                 },
             },
+            checks={
+                "backend-ready": {
+                    "override": "replace",
+                    "level": "ready",
+                    "period": "30s",
+                    "exec": {
+                        "command": (
+                            'bash -c "'
+                            "pebble services backend | grep -q inactive || "
+                            "curl -f http://localhost:6060/readyz"
+                            '"'
```

Comment:
Just out of curiosity, why not just a curl check?
or even something like:
```
    test:
        override: replace
        http:
            url: http://localhost:8080/test
```

Reply:
Under certain conditions, services can be stopped, for instance, when postgresql integration is added but then withdrawn. In such cases, we need to disable the checks as all penpot services will be stopped. Since there is currently no way to stop a pebble check, use this method to ensure the check succeeds if the penpot services are stopped.

Reply:
Should that check be innocuous even it the service is down? If services are down, a ready check will just not route traffic to the pod through the service.

Reply:
Yes, I think you are right; the ready check here should indicate if the service is down in this case.

However, this actually raises another question. Due to the limitation of the `ingress` integration, I am pretty sure that `nginx-ingress-integrator` with `ingress` integration can't use the pod's ready status to redirect traffic. I think this is also the case for `traefik-k8s`. `nginx-ingress-integrator` with `nginx-route` integration can do that though.

We probably need to think about these issues in the future, but for now, I think the best solution is to change this to an alive check.